libmamba/src/api/info.cpp

```diff
@@ -91,9 +81,30 @@ namespace mamba
             Console::instance().json_write(items_map);
         }
 
-        void print_info(Context& ctx, ChannelContext& channel_context, const Configuration& config)
+        void print_info(
+            Context& ctx,
+            ChannelContext& channel_context,
+            const Configuration& config,
+            InfoOptions options
+        )
         {
             assert(&ctx == &config.context());
+
+            if (options.base)
+            {
+                if (ctx.output_params.json)
```

Comment:
Hmm, after looking at the original code, we shouldn't use this condition here (and no `cout` either).
We are using `info_json_print` (for json case) and `info_pretty_print` for regular output where they internally handle the previous condition.
We should modify `items` depending on the `base` flag instead.